book/go/api/api.md

```diff
@@ -66,7 +66,7 @@ Add a stateless computation that only returns one message to the current pipelin
 
 Similar to `To`, but the computation can return more than one message.
 
-##### `ToStatePartition(stateComputation wallarooapi.StateComputation, stateBuilder wallarooapi.StateBuilder, stateName string, partitionFunction wa.PartitionFunction, partitions []uint64) *pipelineBuilder`
+##### `ToStatePartitionWithKeys(stateComputation wallarooapi.StateComputation, stateBuilder wallarooapi.StateBuilder, stateName string, partitionFunction wa.PartitionFunction, partitions []byte) *pipelineBuilder`
```

Comment:
We still have a function called `ToStatePartition(...)`, but it no longer takes the `partition` argument, so we need to document that. Same with `ToStatePartitionMulti(...)`, the function is still there but it no longer takes a `partition` argument. We need to make sure those are documented.

Reply:
Also, the `partition` argument for `ToStatePartitionWithKeys` and `ToStatePartitionMultiWithKeys` is now `[][]byte`, not `[]byte`.